test/e2e/simple.go

```diff
@@ -52,7 +51,7 @@ func E2ESimple(logger *log.Logger) error {
 	testnet.NoError("failed to create tx client", err)
 
 	logger.Println("Setting up testnets")
-	testnet.NoError("failed to setup testnets", testNet.Setup(ctx))
+	testnet.NoError("failed to setup testnets", testNet.Setup(ctx, testnet.WithPrometheus(false))) // TODO: re-enable prometheus once fixed in comet
```

Comment:
any idea what we need to do to remedy this comet situation?
this was causing a panic(?) due to missing labels or something?